apps/web/src/components/AccountCard/AccountBalance.tsx

```diff
@@ -26,37 +30,29 @@ export const AccountBalance = () => {
         >
           Tez Balance
         </Text>
-        <Text color={color("900")} fontWeight="600" size="2xl">
-          {balance && prettyTezAmount(balance)}
-        </Text>
-        <Text color={color("700")} size="sm">
-          {usdBalance && `${usdBalance}$`}
-        </Text>
+        {balance !== undefined && (
```

Comment:
Will it work like `balance && ...` and below too?

Reply:
not exactly. if the balance is 0 then it'll show just 0, not 0 Tez